Dockerfile

```diff
@@ -1,11 +1,20 @@
 # don't change this line
 FROM korney4eg/nginx-php:latest
 
-# your code goes here
-# Make nginx use app.conf configuration to properly render php files
-# Make sure that index.php file is available in browser
-# expose needed ports
+RUN apt update && apt -y upgrade
 
+COPY app.conf /etc/nginx/sites-enabled/app.conf
+RUN chmod +x /etc/nginx/sites-enabled/app.conf
+
+# Uncomment to store index.php inside image
+COPY --chown=www-data index.php /var/www/html/index.php
+RUN chmod +x /var/www/html/index.php
```

Comment:
why do you need execute permissions?

Reply:
Php script needs execute permission because when user runs it via browser, it needs to be executed on a server.

Reply:
Can you test how it behaves without execution permissions?

Reply:
Well it works, I guess execute permission is not necessary.